test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -44,6 +44,7 @@ var (
 	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
 	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
 	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
+	gke            = flag.String("gke", "", "Taget GKE cluster with context gke_{project}_{zone}_{cluster-name}")
```

Comment:
Ah, I didn't realize it defaults to gce. How about renaming the flag to `--gke-context`? That makes it clearer both when to use it and what it's used for.